pom.xml

```diff
@@ -291,6 +291,15 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>JDK11</id>
+      <activation>
+        <jdk>(,11]</jdk>
```

Comment:
The compiler plugin is already executed with JDK 11 or newer, see
https://github.com/google/gson/blob/742fd50ff74350462d8dcf80941efce219e187dd/pom.xml#L123-L125

You should therefore be able to specify `maven.compiler.testRelease` in the main properties, without having to define a separate profile.

Reply:
So, I can put `<maven.compiler.testRelease>11</maven.compiler.release> ` here:
https://github.com/google/gson/blob/742fd50ff74350462d8dcf80941efce219e187dd/pom.xml#L36-L39

and everything should work?

Reply:
Yes I think so. The project already requires JDK 11 to build and fails to compile if you use JDK 8.
However, unfortunately the current error message is a bit cryptic because the Maven plugins do not seem to notice that the used JDK is < JDK 11.

Maybe it would be useful to add the Maven Enforcer Plugin to fail fast when users try to run with < JDK 11, e.g.:
```xml
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-enforcer-plugin</artifactId>
  <version>3.2.1</version>
  <executions>
    <execution>
      <id>enforce-jdk-version</id>
      <goals>
        <goal>enforce</goal>
      </goals>
      <configuration>
        <rules>
          <requireJavaVersion>
            <version>[11,)</version>
          </requireJavaVersion>
        </rules>
      </configuration>
    </execution>
  </executions>
</plugin>
```